packages/desktop-client/src/components/mobile/transactions/TransactionListWithBalances.tsx

```diff
@@ -68,51 +90,61 @@ export function TransactionListWithBalances({
   onLoadMore,
   onOpenTransaction,
   onRefresh,
-}) {
-  const selectedInst = useSelected('transactions', transactions);
+}: TransactionListWithBalancesProps) {
+  const selectedInst = useSelected('transactions', transactions, []);
 
   return (
     <SelectedProvider instance={selectedInst}>
-      <View
-        style={{
-          flexShrink: 0,
-          marginTop: 10,
-        }}
-      >
+      <>
```

Comment:
The only change here is I wrapped the two children in a `<></>` so that there is only a single child inside <Selected Provider>. I did not touch the components inside the `<></>`